ios/BetterSocial.xcodeproj/project.pbxproj

```diff
@@ -2119,8 +2112,7 @@
 				);
 				PRODUCT_BUNDLE_IDENTIFIER = org.reactjs.native.example.BetterSocial;
 				PRODUCT_NAME = "$(TARGET_NAME)";
-				PROVISIONING_PROFILE_SPECIFIER = "match AdHoc org.reactjs.native.example.BetterSocial";
-				"PROVISIONING_PROFILE_SPECIFIER[sdk=iphoneos*]" = "match AdHoc org.reactjs.native.example.BetterSocial";
+				PROVISIONING_PROFILE_SPECIFIER = "";
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `PROVISIONING_PROFILE_SPECIFIER` has been removed. If this is intentional and you're planning to set the provisioning profile specifier elsewhere or manually before each build, then it's fine. Otherwise, this could cause issues with code signing.

```diff
- PROVISIONING_PROFILE_SPECIFIER = "match AdHoc org.reactjs.native.example.BetterSocial";
+ PROVISIONING_PROFILE_SPECIFIER = "";
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->